src/accelerate/components/Dashboard.scss

```
@@ -328,16 +463,52 @@ body.index-php {
 			border-bottom: 1px solid #eee;
 		}
 
-		.table-th-views {
-			min-width: 100px;
-		}
+		.record-thumbnail {
+			width: 110px;
+			padding: 0;
+			padding-right: 40px;
 
-		.table-th-links {
-			min-width: 80px;
+			img {
+				width: 105px;
+				height: 47px;
+				object-fit: cover;
```

Thinking `object-position: center top` as well? You should should remove the width & height from here as it'll screw up the aspect ratio - they belong on the container element this object needs to fit into.

---

`object-position` is a nice addition, the width/height does not screw up the aspect ratio as the `object-fit` property handles positioning the actual image properly.
```suggestion
				object-fit: cover;
				object-position: center top;
```